src/Utils/MockData/BackendMock.js

```diff
@@ -0,0 +1,43 @@
+import Config from './../../config/config.js';
+import Hierarchy from './Transport for London/Hierarchy.js';
+import Entity from './Transport for London/Entity.js';
+
+class BackendMock {
+
+	static getRouteMapper() {
+		return new Map()
+			.set(/entitytype\/hierarchy/i, BackendMock.getEntityTypeHierarchy)
+			.set(/entitytype\/(\d+)\/attributes/i, BackendMock.getEntityTypeAttributes)
+			.set(/entity\/(\d+)/i, BackendMock.getEntity);
+	}
+		
+	static handleRequest(request) {
+		const RouteMapper = BackendMock.getRouteMapper();
+		const cleanedRoute = request.url.replace(Config.backendRESTRoute,"");
+		const targetRoute = [...RouteMapper.keys()].find((route) => {
+			return route.exec(cleanedRoute);
+		});
+		const responseValue = RouteMapper.get(targetRoute)(targetRoute.exec(cleanedRoute));
+		return BackendMock.buildResponse(responseValue);
+	}
+	
+	static getEntityTypeHierarchy() {
+		return Hierarchy;
+	}
+	
+	static getEntity(params) {
+		return Entity.find((entity) => {
+			return entity.Id = params[1];
+		});
+	}
+	
+	static getEntityTypeAttributes(params) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "params". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)